src/dmd/dinterpret.d

```diff
@@ -771,6 +782,8 @@ public:
                 return;
             istate.start = null;
         }
+        if (s.exp && s.exp.hasCode)
+            incUsageCtfe(istate, s.loc);
```

Comment:
Why is this here? Doesn't the next line count any executed lines in s.exp?

Reply:
That would mean every expression would need it's own incUsage call.

Reply:
e2ir has almost no use of `incUsage` because most of em are handled in s2ir.
https://github.com/dlang/dmd/blob/master/src/dmd/s2ir.d#L730